lib/importers/licence_transaction/licence_importer.rb

```diff
@@ -18,8 +25,8 @@ def call
             public_updated_at: licence["public_updated_at"],
             first_published_at: licence["first_published_at"],
             update_type: "major",
-            licence_transaction_location: [],
-            licence_transaction_industry: [],
+            licence_transaction_location: tagging["locations"],
+            licence_transaction_industry: tagging["industries"],
```

Comment:
So does this only add the tagging to new licences? What happens to the existing licences?
Perhaps there needs to be another method that dumps the tagging for existing licences and the re-tags them?

I think that's what was happening before too, so perhaps it's something that was overlooked, or I've forgotten how this works.

Reply:
Currently (and previously) licences that already exist are skipped from the subsequent on a licence by licence basis (https://github.com/alphagov/specialist-publisher/blob/90ce29350e145210038565b23ab8a3227bb62d54/lib/importers/licence_transaction/licence_importer.rb#L43-L46). 

Before we launch, licences get deleted every night in the test environments and so any updates to the tagging file will be applied as the licence will be effectively ‘new’. 

If we wanted to update the code to update existing licences (e.g after we’ve run the import in production) then we’d need to update the code to do that. This feels like something we could do if we think that industry tagging is likely to change on bulk, post launch but that is probably out of scope of this piece of work.

Reply:
Thanks for the reminder.